### PR TITLE
Cleaned up homepage, added links to wiki and emulators

### LINKED ISSUE #10: Error in spelling on website
On the main emudeck.com web site, under the "All your favourite systems" heading, there is an error in spelling for "Super Nintendo" in the block list of emulated systems.

Current spelling on web site is: Super Nntendo

Ostensibly, this should be: Super Nintendo

### PR DESCRIPTION
**Pull request by ghost**
Fixes: https://github.com/EmuDeck/EmuDeckWebsite/issues/10

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2059,11 +2060,27 @@ <h2>Play all of your retro games on the go</h2>
             <img src="img/emulators.jpg" />
           </div>
           <div class="col_6">
-            <h2>All your favourite systems</h2>
+            <h2>All of your favourite emulators</h2>
             <p class="large">
-              Atari, Genesis/Mega Drive , Sega CD, Sega 32X, PC Engine, NES, Super Nntendo , MAME, FinalBurn Neo, Master
-              System, Game Boy, Neo Geo Pocket, Game Gear, Nintendo DS, Nintendo 3DS, Sony PSP, Dreamcast, Playstation,
-              Nintendo 64, Wii, GameCube, Wii U, Switch, etc.
+            <ul>
```
Comment (dragoonDorise) [inline index.html]: @rawdatafeel I prefer to keep this clean, please remove the ul and left the previous list :)

File: index.html [hunk 2]
```diff
@@ -2073,10 +2090,10 @@ <h2>All your favourite systems</h2>
       <div class="content">
         <div class="gridrow">
           <div class="col_6">
-            <h2>Standard Hotkeys</h2>
+            <h2>Hotkeys Galore</h2>
             <p class="large">
-              We follow the AmberElec convention so you can jump right in and know how to save games. And in some
-              emulators we unleash all the Steam Deck's extra controls using custom Steam Input Profiles.
+              EmuDeck follows the AmberElec convention so you can jump right in and button combo like an expert. In some
+              emulators, we unleash all the Steam Deck's extra controls using custom Steam Input Profiles. For a full list of the hotkeys EmuDeck utilizes, visit the <a target="_blank" href="https://github.com/dragoonDorise/EmuDeck/wiki/Hotkeys">hotkey page</a> on the EmuDeck Wiki.
```
Comment (dragoonDorise) [inline index.html]: @rawdatafeel please remove all links in the descriptions, I don't want people to get too much information and get lost in the process

File: index.html [hunk 3]
```diff
@@ -2114,9 +2129,8 @@ <h2>Easy to install but flexible</h2>
           <div class="col_6">
             <h2>Beautifully detailed</h2>
             <p class="large">
-              Your games will be displayed with box art alongside the rest of your Steam library, thanks to a powerful
-              tool called Steam ROM Manager. We will download and configure Steam ROM Manager for you, so everything is
-              done with just a single click.
+              Add your games with box art alongside the rest of your Steam library, thanks to a powerful
+              tool called <a target="_blank" href="https://github.com/SteamGridDB/steam-rom-manager">Steam ROM Manager.</a> EmuDeck downloads and configures Steam ROM Manager for you. Add your ROMs to your Steam Library with just a single click.
```
Comment (dragoonDorise) [inline index.html]: @rawdatafeel  same, remove link

File: index.html [hunk 4]
```diff
@@ -2134,9 +2148,7 @@ <h2>Beautifully detailed</h2>
           <div class="col_6">
             <h2>EmulationStation Desktop Edition</h2>
             <p class="large">
-              Do you prefer to keep your ROMs separated from your Steam library? We've got you covered! Installing
-              EmuDeck will also carry over our configurations into EmulationStation Desktop Edition, greatly simplifying
-              the setup process.
+             Use <a target="_blank" href="https://es-de.org/">EmulationStation Desktop Edition</a> to manage and launch your ROMs. A beautiful frontend with endless customization and themes.
```
Comment (dragoonDorise) [inline index.html]: @rawdatafeel same, link out

File: index.html [hunk 5]
```diff
@@ -2177,25 +2187,34 @@ <h2>Install guide</h2>
...
             <li>
-              If you plan to store your roms on a SD card...
-              <strong>Format your SD Card in Steam UI. SD Cards need to be on ext4 (or btrfs) to work on EmuDeck</strong
-              >. Then go into Desktop mode by pressing the STEAM button, Power -> Switch to Desktop
+              To use an SD Card,
+              <strong>format your SD Card in Game Mode on SteamOS. SD Cards need to be formatted as ext4 (or btrfs) to be compatible with EmuDeck</strong>.
             </li>
-            <li>Download the Installer down below, copy that file to your Deck's Desktop and run it.</li>
-            <li>When The installation finish you'll need to copy yor games and run an app called Steam Rom Manager.</li>
+            <li>Switch to Desktop mode. To switch to Desktop Mode, press the STEAM button, then the Power button, and select Switch to Desktop.</li>
```
Comment (dragoonDorise) [inline index.html]: @rawdatafeel I understand you wanted to make it all really clear, but now its such a long list that's going to intimidate people. People is just going to follow the video, the list is more to give a feeling of "wow, only 5 steps, that's easy!" I'm ok if you change those five steps to make it more clear or fix some grammar, but please keep it short :)

**Comment by dragoonDorise** [comment]
@rawdatafeel I've give you some feedback, please take a look whenever you can :)

File: index.html (hunk 3 above)
Comment (ghost) [reply index.html]: Sorry! Removed.

File: index.html (hunk 5 above)
Comment (ghost) [reply index.html]: How is this? 

image

File: index.html (hunk 1 above)
Comment (ghost) [reply index.html]: @dragoonDorise Moved to a credits section, let me know your thoughts on that. 

image

**Pull request merged**